bindata/assets.go

```diff
@@ -0,0 +1,24 @@
+package v410_00_assets
```

Comment:
do we still need this package name?

Reply:
it's mostly done for backward compatibility. If you think it is reasonable to change it to something else, I'm absolutely fine with it.